src/Compilers/Test/Utilities/VisualBasic/ParserTestUtilities.vb

```diff
@@ -539,38 +541,41 @@ Public Module VerificationHelpers
         End Function
     End Class
 
-    Friend Sub AppendSyntaxErrors(errors As IEnumerable(Of Diagnostic), output As StringBuilder)
+    Friend Sub AppendSyntaxErrors(errors As IEnumerable(Of Diagnostic), output As PooledStringBuilder)
```

Comment:
> >    Not changing it, will result in a cast.    
>
> Could you elaborate, please?

`PooledStringBuilder` doesn't inherit from `StringBuilder` but it has Widening Cast operator to `StringBuilder`